eZ Platform/APIs/GraphQL/Requirements.md

```diff
@@ -0,0 +1,37 @@
+# Query Language API Prototype Requirements
```

Comment:
For info: this is the consolidated specs/requirement for GraphQL, Views and GraphQL in the native PHP API.